README.md

```diff
@@ -276,6 +276,23 @@ A configuration that can be passed to the [bell][bell] OAuth module to authentic
 #### Expected Promise Response
 1. Resolve with a valid [bell][bell] configuration
 
+### getOpenedPRs
+The parameters required are:
+
+| Parameter        | Type  | Required | Description |
+| :-------------   | :---- | :------- | :-------------|
+| config        | Object | true | Configuration Object |
+| config.scmUri | String | true | The scm uri (ex: `github.com:1234:branchName`) |
+| config.token | String | true | Access token for scm |
+
+#### Expected Outcome
+The list of opened PR names for the repository. For example: ['PR-5', 'PR-6']
```

Comment:
Technically it needs to be a list of open PR names for the branch, not just repository.